conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinPersistenceTest.scala

```diff
@@ -0,0 +1,62 @@
+package tech.cryptonomic.conseil.common.bitcoin
+
+import scala.concurrent.ExecutionContext
+
+import cats.effect._
+import cats.arrow.FunctionK
+import org.scalatest.{Matchers, WordSpec}
+import slickeffect.Transactor
+import slick.jdbc.PostgresProfile.api._
+import slick.dbio.DBIO
+
+import tech.cryptonomic.conseil.common.testkit.InMemoryDatabase
+
+class BitcoinPersistenceTest
+    extends WordSpec
+    with Matchers
+    with InMemoryDatabase
+    with BitcoinInMemoryDatabaseSetup
+    with BitcoinFixtures {
+
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
+
+  "Bitcoin persistence" should {
+
+      "write blocks" in new BitcoinPersistanceFixtgures {
+        (for {
```

Comment:
Maybe having a named value assigned to the result of this operation could improve the test readability?
Also, the parens seems unnecessary?

Reply:
Actually, parentheses are required, I wrap for comprehension to the `unsafeRunSync`

Reply:
I've refactored the tests, please check it again